### ISSUE TITLE
Why PyHAT , when PHAT Sounds perfect.

### ISSUE THREAD

**Issue by v3ss0n** [PyHAT-stack/awesome-python-htmx#15]
Would be funny too.

**Comment by tataraba** [comment]
Yes, someone actually mentioned this in one of the discussions. I've used PyHAT in a few talks and other docs, but I think it was always initially up for discussion. I like the visual of PyHAT with logo potential (snake with a hat 🐍 🤠 ),  but I agree that PHAT stack _sounds_ better.

@benjamin-kirkbride any thoughts?

**Comment by benjamin-kirkbride** [comment]
I would like to prefice everything I'm about to say with the disclaimer that I do not believe that being fat is wrong or a moral failing. Everything I think about and talk about below is me trying to explain how I think that decisions we make might harm our project, and harm others.

I also want to make it clear that I don't believe anyone advocating for "PHAT" is doing so with bad intentions, for what that's worth.

---

The reason that I have been opposed to "PHAT" in the past is because it is a homophone of "fat" and often used in slang to mean "fat". I do not want this project to be mired with accusations of fat-phobia, ablism, etc. [It can kill a project when the name is controversial.](https://www.reddit.com/r/selfhosted/comments/14pdu0m/introducing_crackpipe_your_decentralized/)

As far as the wold "phat" appearing in dictionaries to mean something else, here is what [etymonline.com has to say](https://www.etymonline.com/word/phat):

> hip-hop slang, "great, excellent," 1992, originating perhaps in the late 1980s and meaning at first "sexiness in a woman." The word itself is presumably a variant of fat (q.v.) in one of its slang senses, with the kind of off-beat spelling preferred in street slang (compare boyz). The spelling is attested as far back as 1678, as an erroneous form of fat (a classical over-correction; see ph).
> 
> In the modern word it is said by some to be an acronym, and supposed originals are offered: "pretty hot and tasty," or "pretty hips and thighs" among them, all unconvincing. These may have begun as improvised explanations to women who felt insulted by the word.

This is more or less what I thought it meant. I realize that the word "phat" does not have the exact same negative connotation as "fat" does. That said:
- this is typically aimed at women (and seemingly also typically aimed at women of color), and seemingly it is aimed at women who do not appreciate being called that (associated with catcalling?)
-  I could easily see this being considered a form of benevolent prejudice, which is something that has become more widely known to be bad in recent years.

There are a lot of adjectives that are in theory "good", but have a history of being used by men to catcall, degrade, objectify, etc women. Many of these words also have a history of racial prejudices as well.

Examples:
- curvy
- thicc
- spicy
- foxy / vixen
- juicy
- chick

I think "phat" is also on this list. All that being said, I don't think that the word PHAT likely to cause harm to anyone, which is obviously the most important thing. But I do think that it may cause harm to the project. I agree it is a more catchy name, but PyHAT is pretty good too IMO.

It is hard for me to say "I think using PHAT is fine" because I am not fat, nor a woman, nor am I black (nor am I in the hip-hop scene).

Again I want to make it clear that I don't believe anyone advocating for "PHAT" is doing so with bad intentions.

**Comment by v3ss0n** [comment]
Why? This is veering too much from official definition from various leading dictionary. I had looked into top Three dictionary and all return non sexist, or nothing related to fat.  I am 6 ft 250lbs and I am not offended by that.

**Comment by benjamin-kirkbride** [comment]
> This is veering too much from official definition from various leading dictionary

@v3ss0n dictionaries are for definitions, but they typically do not make moral judgement. One must consult etymological sources to determine the history, context, and connotation associated with a given word.

It's also worth mentioning that dictionaries are not the arbiters of "what words mean"; words mean what people intend and understand them to mean, and dictionaries aim to capture that, but dictionaries tend to not be great at accurately defining slang, and [education](https://en.wikipedia.org/wiki/African-American_Vernacular_English_and_education)/[academia](https://en.wikipedia.org/wiki/African-American_Vernacular_English#In_academia) in general are notoriously discriminatory towards AAVE, so again discernment is required when researching these topics.

I consulted with the following sources in researching the above:
- https://www.etymonline.com/word/phat
- https://www.snopes.com/fact-check/chewing-the-phat/
- https://en.wiktionary.org/wiki/phat#Etymology

In addition, I spoke with a woman who was into Hip Hop in the 80's when in re-surged in popularity in AAVE.
